app/src/main/java/com/example/umlandowallet/ui/SettingsScreen.kt

```diff
@@ -11,7 +11,9 @@ import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import androidx.navigation.NavController
+import com.example.umlandowallet.Global
 import com.example.umlandowallet.R
+import org.bitcoindevkit.AddressIndex
 
 @Composable
 fun SettingsScreen(navController: NavController) {
```

Comment:
I forgot to add it to my last PR but if you want to make the column scrollable you can add the `.verticalScrolll()` modifier to the column. Your whole modifier would look like this:
```kotlin
Column(
    modifier = Modifier
        .padding(top = 48.dp)
        .fillMaxSize()
        .verticalScroll(rememberScrollState())
) {
    // ...
}
```